game/plugins/src/main/kotlin/gg/rsmod/plugins/api/ext/IntExt.kt

```diff
@@ -35,4 +35,29 @@ fun Int.toLiteral() : String? {
     }
 
     return literal
+}
+
+/**
+ * Gets the 'interface hash' of a given interface id and
+ * child component.
+ *
+ * @param child     The child component
+ */
+fun Int.getInterfaceHash(child: Int = -1) : Int {
```

Comment:
I'm not quite sure if this belongs here. It would maybe fit somewhere else (interface related), or a separate (perhaps inline) class be created for handling raw interface values such as this.